_rules/SC1-2-video-transcript.md

```diff
@@ -21,7 +21,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video contains audio.
 
 ### Expectation 1
 
```

Comment:
Wonder if expectation 1 & 2 should be combined - the "from expectation 1" cross reference seems kind of procedural, and should be avoided IMO

Reply:
I think my solution works. Thanks for the suggestion!